plugins/infinispan90/pom.xml

```diff
@@ -124,6 +124,17 @@
          <optional>true</optional>
       </dependency>
 
+      <dependency>
+         <groupId>org.infinispan</groupId>
+         <artifactId>infinispan-marshaller-kryo</artifactId>
+         <version>${version.infinispan}</version>
```

Comment:
Alright. The Kryo and Protostuff marshaller is part of the main repository. So that explains it.